cw/__init__.py

```diff
@@ -20,35 +22,62 @@
           'rabbit-exchange': os.getenv('LCW_RABBIT_EXCHANGE', 'local.lcmap.changes.worker'),
           'rabbit-result-routing-key': os.getenv('LCW_RABBIT_RESULT_ROUTING_KEY', 'change-detection-result'),
           'rabbit-ssl': os.getenv('LCW_RABBIT_SSL', False),
-          'ubid_band_dict' : {
+          'api-host': os.getenv('LCW_API_HOST', 'http://localhost:5678'),
+          'ubid_band_dict': {
               'tm': {'red': 'band3',
                      'blue': 'band1',
                      'green': 'band2',
                      'nirs': 'band4',
-                     'swirs1': 'band5',
-                     'swirs2': 'band7',
+                     'swir1s': 'band5',
+                     'swir2s': 'band7',
                      'thermals': 'band6',
                      'qas': 'cfmask'},
               'oli': {'red': 'band4',
                       'blue': 'band2',
                       'green': 'band3',
                       'nirs': 'band5',
-                      'swirs1': 'band6',
-                      'swirs2': 'band7',
+                      'swir1s': 'band6',
+                      'swir2s': 'band7',
                       'thermals': 'band10',
                       'qas': 'cfmask'}}}
 
+numpy_type_map = {
+    'UINT8': np.uint8,
+    'UINT16': np.uint16,
+    'INT8': np.int8,
+    'INT16': np.int16
+}
+
+
+def spectral_map(cfg):
```

Comment:
this probably belongs in spark.py or in a module sitting next to it.  It's weird to have calls going two directions, from cw to spark and from spark to cw.